packages/css/src/components/fieldset/fieldset.scss

```diff
@@ -0,0 +1,37 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/hyphenation";
+
+@mixin reset {
+  border: none;
+  margin-inline: 0;
+  padding-block: 0;
+  padding-inline: 0;
+}
+
+.amsterdam-fieldset {
+  border: none;
+  margin-inline: 0;
+  padding-block: 0;
+  padding-inline: 0;
+}
+
+@mixin legend-reset {
+  padding-inline: 0;
+  -webkit-text-size-adjust: 100%;
+}
```

Comment:
This is only used once. Why make it reusable?

Reply:
We have an undocumented convention to wrap reset rules in a mixin, to show the difference between reset rules and true styling rules. We should really document this, I'll make a note of it